firmware/library/L1_Drivers/ssp.hpp

```diff
@@ -0,0 +1,206 @@
+/**
+ ** Provide How-to for driver
+
+ **
+ **
+ */
+
+#pragma once
+
+#include "LPC40xx.h"
+#include "pin_configure.hpp"

```

Comment:
Fully qualify the path with respect to the library folder. For example, this should be "L1_Driver/pin_configure.hpp"